src/plugin/getErrorTrackingPluginsFromOptions.ts

```diff
@@ -62,5 +73,10 @@ export const getErrorTrackingPluginsFromOptions = (
     ) as (keyof FileUploadOptions)[]
   ).filter((option) => !options || options[option] !== false);
 
-  return configPluginsKeys.map((key) => ERROR_TRACKING_CONFIG_PLUGINS_MAP[key]);
+  return [
+    withAndroidGradleConfig({
+      serviceName: options?.serviceName,
+    }),
+    ...configPluginsKeys.map((key) => ERROR_TRACKING_CONFIG_PLUGINS_MAP[key]),
+  ];
```

Comment:
After testing this we actually need to put the `withAndroidGradleConfig` plugin last. 
The reason is that plugins are executed in order, and then all callbacks passed to mods such as `withAppBuildGradle` are executed in reverse order 🤯 
So in order to apply it first, we should put it last here.

```suggestion
  return [
    ...configPluginsKeys.map((key) => ERROR_TRACKING_CONFIG_PLUGINS_MAP[key]),
    withAndroidGradleConfig({
      serviceName: options?.serviceName,
    }),
  ];
```

Reply:
Interesting. Learn something new every day!

Thanks for the help testing this. Updated! 🚀